build/__dotNetCoreRunTimeVersions.sh

```diff
@@ -1,24 +1,24 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-DOT_NET_CORE_RUNTIME_BASE_TAG='20210201.3'
+DOT_NET_CORE_RUNTIME_BASE_TAG='20210210.1'
```

Comment:
I think it'll be updated to today's tag `20210216.2`

Reply:
yes I will update it again